packages/tasks/src/tasks/feature-extraction/inference.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+/**
+ * Inputs for Feature Extraction inference
+ */
+export interface FeatureExtractionInput {
+	/**
+	 * One or several texts to get the features of
+	 */
+	inputs: string[] | string;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: { [key: string]: unknown };
+	[property: string]: unknown;
```

Comment:
Text Embedding Inference supports both `truncate` and `normalize` boolean parameters (see [here](https://huggingface.github.io/text-embeddings-inference/#/Text%20Embeddings%20Inference/embed)). It has been asked twice in `huggingface_hub` but for now I'm waiting for the "official" spec before adding them. Should we add them now (I'm in favor of it since TEI should become the "flagship" for embeddings)

Reply:
None of this would work out of the box in `sentence_transformers` API, but I guess we can add later on if needed